### ISSUE TITLE
1.4.6 GUI, Policy # feedback/opinion/suggestion on GUI

### ISSUE THREAD

**Issue by beyo-ra** [veraPDF/veraPDF-apps#198]
Some feedback from the training seminar in Stockholm (20170529),
especially concerning the GUI for policy.

- Introduce condition "ignore", that is, if the tag is present, ignore it and accept whatever value it has.
- Introduce condition and/or relationship between policy rules.

Furthermore, the relationship between "Configs/Features Config" for standalone use and for policy use is not intuitive.

I would recommend
- allowing activation/deactivation of the Features Config options to be available during Policy Config, 
- and somehow distinguish extraction of data from PDF, and include activation/disabling Features Config options, as and under the term "feature extraction", "data extraction" or such.

**EXAMPLE config/policy START**

```
1. Initial state
-- Either for all features or for example add those required using a dropbox.

[ ] Annotation -----+
+-------------------+

2. Activate feature report for Annotation

[x] Annotation -----+
| add rule...       |
+-------------------+

3. Add rule 1

[x] Annotation -----+
| [ ] rule 1        |
| add rule...       |
+-------------------+

4. Activate rule 1
5. Create rule 2 (but not activated)

[x] Annotation -----+
| [x] rule 1        |
| [ ] rule 2        |
| add rule...       |
+-------------------+
```

**EXAMPLE config/policy STOP**

**And consider...**
... changing config/feature to e.g. config/extraction

and, when selecting report type "feature" in the drop box (amongst validation, validation & feature, policy) activate a popup dialog with the available selection currently showed in "config/feature".

**Comment by bitsgalore** [comment]
I agree with the above comment. In addition to this, I don't think the *Configs* menu is the most obvious place for this. In my case I was completely unaware of the policy creation GUI until I saw it mentioned [here](https://github.com/veraPDF/veraPDF-apps/issues/197#issuecomment-306152289). 

Also, I don't understand why the policy file is called as a *policy config file* in the save dialog (what's the difference between a policy *config* file and a *policy* file? Is there any?).  

Finally, the icon you need to click on to add a new policy rule (button with three dots, "...") is rather counter-intuitive, I actually expected this to activate a file open dialog! Why not replace this with a "+" sign?
